src/Components/Cards/HorizontalProductsCard.jsx

```diff
@@ -0,0 +1,65 @@
+import { useWishlist } from "../../Context/WishlistContext";
+import { useCart } from "../../Context/CartContext";
```

Comment:
You can export it in ` index.js ` file and import from there.

Reply:
Thank you for your suggestions, will do it in the next iteration.